### PR TITLE
Clothing Commits 2 Electric Boogalo

### PR DESCRIPTION
**Pull request by thestalker12**
## Changelog
rscadd: Clothing to the textile machine 
/:cl:

 Starting with Hard Hats, not ready for merge

### CODE REVIEW

File: code/modules/clothing/head/hardhat.dm [hunk 1]
```diff
@@ -21,6 +21,9 @@
 	heat_protection = SLOT_HEAD
 	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 	max_pressure_protection = FIRESUIT_MAX_PRESSURE
+	material = /decl/material/solid/metal/steel
+	matter = list(/decl/material/solid/metal/plasteel = MATTER_AMOUNT_TRACE)
```
Comment (comma) [inline code/modules/clothing/head/hardhat.dm]: plasteel seems a bit too costly for something so wide-spread, and it doesn't even have that good armor, steel-tier values.
heck I'd even go for plastic with steel reinforcment.

File: code/modules/clothing/head/_head.dm [hunk 2]
```diff
@@ -12,7 +12,9 @@
 	var/light_applied
 	var/brightness_on
 	var/on = 0
-
+	origin_tech = "{'materials':1,'engineering':1}"
+	material = /decl/material/solid/cloth
```
Comment (MistakeNot4892) [inline code/modules/clothing/head/_head.dm]: `origin_tech` and `material` are already set to these values on `/obj/item/clothing`, they don't need to be set again.

File: code/modules/clothing/head/hardhat.dm (hunk 1 above)
Comment (thestalker12) [reply code/modules/clothing/head/hardhat.dm]: Ill fuck with this later, Truthfully I don't have the energy right now given I have classes starting in like 4 days

File: code/modules/clothing/head/_head.dm (hunk 2 above)
Comment (thestalker12) [reply code/modules/clothing/head/_head.dm]: Ill changed this out later

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**